src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/InterfaceMethodsShouldBeCallableByChildTypes.cs

```diff
@@ -73,7 +73,8 @@ private static bool ShouldExcludeOperationBlock(ImmutableArray<IOperation> opera
                 }
 
                 if (block.Statements.Length == 0 ||
-                    (block.Statements.Length == 1 && block.Statements[0].Kind == OperationKind.ThrowStatement))
+                    (block.Statements.Length == 1 &&
+                     (block.Statements[0] as IExpressionStatement)?.Expression.Kind == OperationKind.ThrowExpression))
```

Comment:
Perhaps break this up with `is` expressions? `block.Statements[0] is IExpressionStatement exprStatement && exprStatement.Expression.Kind == OperationKind.ThrowExpression`

Reply:
Sure, will change.